src/GeositeFramework/Views/Home/Index.cshtml

```diff
@@ -384,8 +384,12 @@
                             <i class="icon-info-circled-1"></i>
                         </button>
                     <% } %>
-
-                    <button class="button-link plugin-off i18n" data-i18n="[title]Turn off this plugin" title="Turn off this plugin"><i class="icon-cancel"></i></button>
+                    <button class="button-link plugin-minimize i18n" data-i18n="[title]Minimize this plugin" title="Minimize this plugin">
+                        <i class="icon-minimize"></i>_
+                    </button>
+                    <button class="button-link plugin-off i18n" data-i18n="[title]Turn off this plugin" title="Turn off this plugin">
```

Comment:
Since you've added the i18n stuff here, can you add the text and a google translation of spanish into:

https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/master/src/GeositeFramework/locales/en.json
https://github.com/CoastalResilienceNetwork/GeositeFramework/blob/master/src/GeositeFramework/locales/es.json